MdeModulePkg/MdeModulePkg.dec

```diff
@@ -442,9 +442,14 @@
   #
   gAdvLoggerAccessGuid = { 0xa021bf2b, 0x34ed, 0x4a98, {0x85, 0x9c, 0x42, 0x0e, 0xf9, 0x4f, 0x3e, 0x94 }}
 
+  #
+  # Guids for NVMe Timeout Events
+  # {4754469d-6528-4dfc-84aa-8c8a03a2158b}
+  gNVMeEnableStartEventGroupGuid = { 0x4754469d, 0x6528, 0x4dfc, {0x84, 0xaa, 0x8c, 0x8a, 0x03, 0xa2, 0x15, 0x8b}}

```

Comment:
It is unfortunate this didn't follow the case convention to be `gNvmeEnableStartEventGroupGuid`. Since it is a public facing name, I don't think it's worth changing now.